ios/airship.podspec

```diff
@@ -16,7 +16,7 @@ A new flutter plugin project.
   s.public_header_files = 'Classes/**/*.h'
   s.dependency 'Flutter'
 
-  s.ios.deployment_target   = "10.0"
-  s.dependency              'UrbanAirship-iOS-SDK', '~> 11.0.0'
+  s.ios.deployment_target   = "11.0"
+  s.dependency              'UrbanAirship-iOS-SDK', '~> 12.0.0'
```

Comment:
should we make this 12.0.1, now that it's out?

Reply:
yes we should